src/TideSearchOperation.php

```diff
@@ -122,9 +122,14 @@ public function createSearchListingContentType(array $config_location) {
       $storage = \Drupal::entityTypeManager()->getStorage($type);
       $id = substr($config, strrpos($config, '.') + 1);
       if ($storage->load($id) == NULL) {
-        error_log(" tide_search - importing config: " . $id);
-        $config_entity = $storage->createFromStorageRecord($config_read);
-        $config_entity->save();
+        // Try using an extended id as well, e.g.
+        // field.storage.paragraph.field_header_configuration will try paragraph.field_header_configuration
+        $id = substr($config, strrpos($config, '.', strrpos($config, '.') - strlen($config) - 1) + 1);
```

Comment:
Sorry for obscure command, but basically it sets the offset on the `strrpos` to start just left of where the first "." is found in order to find the second last ".".

Reply:
New content types in the future should probably be a sub module with proper namespaced storage to not clash. Lesson learnt!

Reply:
Wait a minute, I didn't even add you as a reviewer! Go away :D

Reply:
A matter of personal taste perhaps, but I think this be more readable splitting the string into an array and working with that.

```
$parts = explode(".", $config);
$id = $parts[count($parts)-2];
```

Reply:
Yeah, agree. Hack fix to get it over the line in this instance.